src/file-upload/internal.tsx

```diff
@@ -134,7 +135,8 @@ function InternalFileUpload(
             multiple={multiple}
             onChange={event => handleFilesChange(event.detail.value)}
             value={value}
-            {...restProps}
+            ariaLabelledby={restProps.ariaLabelledby}
+            controlId={restProps.controlId}
```

Comment:
What is the reason of having these 2 props passed down explicitly?

Reply:
It was picking up className and id, and then a test was failing that className & id should only apply to a single component. We don't want className to apply to both the file input and file upload wrappers